ui/components/institutional/compliance-settings/compliance-settings.js

```diff
@@ -0,0 +1,98 @@
+import React, { useContext } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  JustifyContent,
+  DISPLAY,
+  TextColor,
+  FLEX_DIRECTION,
+} from '../../../helpers/constants/design-system';
+import { I18nContext } from '../../../contexts/i18n';
+import { mmiActionsFactory } from '../../../store/institutional/institution-background';
+import { Text } from '../../component-library';
+import Box from '../../ui/box';
+import Button from '../../ui/button';
+
+const ComplianceSettings = () => {
+  const t = useContext(I18nContext);
+  const dispatch = useDispatch();
+  const mmiActions = mmiActionsFactory();
+
+  const complianceActivated = useSelector((state) =>
+    Boolean(state.metamask.institutionalFeatures?.complianceProjectId),
+  );
+
+  const disconnectFromCompliance = async () => {
+    await dispatch(mmiActions.deleteComplianceAuthData());
+  };
+
+  const renderDisconnect = () => {
+    return (
+      <Button
+        type="default"
+        large
+        onClick={disconnectFromCompliance}
+        data-testid="disconnect-compliance"
+      >
+        {t('disconnect')}
+      </Button>
+    );
+  };
```

Comment:
Don't do this, just render the Button inline. same with the next Button.

Reply:
OK.